packages/desktop-ui-lib/src/lib/permission-manager/permission-manager.component.html

```
@@ -252,6 +262,62 @@ <h6>{{ 'TIMER_TRACKER.PERMISSIONS.ACCESSIBILITY' | translate }}</h6>
+				<nb-card>
+					<nb-card-body>
+						<div class="row d-flex align-items-center mb-3">
+							<div class="col-6">
+								<span class="subtitle-tile">
+									{{ 'TIMER_TRACKER.PERMISSIONS.HARDWARE_ACCELERATION_STATUS' | translate }}
+								</span>
+							</div>
+							<div class="col-6 d-flex justify-content-end">
+								@if (hardwareAccelerationDisabled()) {
+									<button
+										nbButton
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)